ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-custom/query-custom.view.js

```diff
@@ -65,6 +67,14 @@ define([
             this.model.set({
                 cql: filter
             });
+        },
+        serializeData: function() {
+            var templateTitle = this.model.get('name') != null
+                                 ? this.model.get('name')
+                                 : "Standard Search";
```

Comment:
Why `Standard Search`?

Reply:
May actually remove the label altogether, so this may no longer be necessary, wdu think? We have a label at the top of the window already, so might seem redundant.

Reply:
sgtm